cmd/logger.go

```diff
@@ -136,7 +155,26 @@ func getSource() string {
 
 	return fmt.Sprintf("[%s:%d:%s()]", filename, lineNum, funcName)
 }
-
+func printJSON(fatal bool, source, cause, stack, msg string, data ...interface{}) {
+	level := "Error"
+	if fatal {
+		level = "Fatal"
+		defer os.Exit(1)
+	}
+	log := &logEntry{
+		Level:   level,
+		Source:  source,
+		Stack:   stack,
+		Cause:   cause,
+		Time:    time.Now().UTC().String(),
+		Message: fmt.Sprintf(msg, data...),
+	}
+	buf := new(bytes.Buffer)
+	enc := json.NewEncoder(buf)
+	enc.SetEscapeHTML(false)
+	_ = enc.Encode(log)
+	fmt.Println(string(buf.String()))
```

Comment:
You don't need to convert into string twice, i.e.
`fmt.Println(string(buf.String()))` => `fmt.Println(buf.String())`